xgi/linalg/matrix.py

```diff
@@ -248,9 +254,12 @@ def laplacian(H, order=1, rescale_per_node=False, index=False):
     .. [1] Lucas, M., Cencetti, G., & Battiston, F. (2020).
         Multiorder Laplacian for synchronization in higher-order networks.
         Physical Review Research, 2(3), 033410.
-    """
 
+    """
     A, row_dict = adjacency_matrix(H, order=order, weighted=True, index=True)
+    if A.shape == (0,):
```

Comment:
similar problem here i think

Reply:
In this case I think it's actually fine, because even when the adj matrix is all zeros, you can compute the Laplacian in the same way as before, so you only need to check whether it's empty.